### ISSUE TITLE
Tracker has stopped working entirely for slowly growing majority of players 👀

### ISSUE THREAD

**Issue by eChef** [ShootMe/FallGuysStats#117]
Starting a few days ago,
I've seen an increasing number of fellow PC Fall Guys players/tracker users, reporting that their trackers have stopped working permanently, no matter what they try: resetting, reinstalling, etc.
And this seems to be affecting a growing number of tracker users.., if not the majority by now 🤔

[Example Tweet with replies about tracker not working 😅](https://twitter.com/Cimarus2/status/1511779521611149318)

Thank you for developing the tracker! 😻😅

**Comment by ShootMe** [comment]
Interesting, I don't play Fall Guys really anymore so there must have been an update or something that broke functionality in the logging? Ill try and take a look today if I can find some time.

**Comment by RainbowStays** [comment]
The tracker works well for solo play, but stops working for party.

**Comment by ExodiaRM** [comment]
> Interesting, I don't play Fall Guys really anymore so there must have been an update or something that broke functionality in the logging? Ill try and take a look today if I can find some time.

The problem only happens if you in a group from the lobby, otherwise the trackers works properly; Begin to happen since the new event Adapt was introduce and for ps4 players a new launcher became available

**Comment by gutsymajortomato** [comment]
also having the same issue.

**Comment by ShootMe** [comment]
I wasn't able to see the issue but was only able to do solo play, which sounds like it's fine. If any of you guys can send me a Player.log file (Found in C:\Users\(Your User)\AppData\LocalLow\Mediatonic\FallGuys_client\) where you did group play that doesn't work with the tracker I can try and use that to debug the issue.

**Comment by wolfchild59** [comment]
> I wasn't able to see the issue but was only able to do solo play, which sounds like it's fine. If any of you guys can send me a Player.log file (Found in C:\Users\ [Your User] \AppData\LocalLow\Mediatonic\FallGuys_client) where you did group play that doesn't work with the tracker I can try and use that to debug the issue.

If you have a secure place to send the logs I can send mine. Both last night and tonight I played solo and while in a party. even testing if it made a difference being in a party vs. being the party leader and main show vs. squads while in a party to see if it made any difference in what was picked up by the tracker or not.

**Comment by ShootMe** [comment]
You can dm it to me on twitter @D3vilSquirrel if you don't want to upload it here.

**Comment by adamkelly86** [comment]
I have an example of where it is failing in squads.   Starts working again as soon as I got eliminated 

Got the log stored and happy to DM you a copy

**Comment by adamkelly86** [comment]
Unloading 1 Unused Serialized files (Serialized files now loaded: 38)
20:12:59.106: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=3 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.106: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=8 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.106: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=13 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.106: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=16 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.112: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=17 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.124: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=18 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
UnloadTime: 11.350900 ms
20:12:59.243: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=19 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:12:59.243: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=20 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=

Unloading 234 unused Assets to reduce memory usage. Loaded Objects now: 155899.
Total: 184.011400 ms (FindLiveObjects: 6.588200 ms CreateObjectMapping: 4.490400 ms MarkObjects: 171.332300 ms  DeleteObjects: 1.599100 ms)

20:13:00.204: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=20 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:13:01.221: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=20 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:13:02.221: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=20 randSeed=497698489, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:13:03.154: Received NetworkGameOptions from 212.78.94.166:8177: NetworkGameOptions: durationInSeconds=120 roundID=round_iceclimb_squads matchmakingBucketID=6_toms_low loadingScreen=Splash_FreezyPeak playerNameMaxDistance=50 minParticipantsToStart=40 maxParticipantsToStart=60 currentParticipantCount=20 randSeed=1983452928, useVariants=True, teamCount=0, UseTeamDistributionRatios=FalsequalificationPercentage=100, squadsQualificationPercentage=66, endRoundIfEnoughTeamsQuit=False, isFinalRound=False, TeamMode=Solo, RequiredTeamEliminations=0, IsScoringGame=False, ScoreTarget=0, , IsScoreTargetDynamic=False, UseTeamDynamicScoreTargetMultipliers=False, ScoreTargetSquad2Players=0,  ScoreTargetSquad3Players=0,  ScoreTargetSquad4Players=0, GameManagerSpawnType=NoneOvertimeMode=None, OvertimeAmount=0, RoundEndCondition=SuccessQuota, RoundEndLeniencyMs=0, UseTeamIdxWinsOnTimeElapsed=False, TeamIdxWinsOnTimeElapsed=0RoundIndex=4, UseShouldHideOpposingTeamNameTags]False ShouldHideOpposingTeamNameTags=
20:13:03.971: [StateWaitingForGameToStart] Server is asking us to get ready for the game - Switching to loading game state.
20:13:03.971: [GameStateMachine] Replacing FGClient.StateWaitingForGameToStart with FGClient.StateGameLoading

**Comment by adamkelly86** [comment]
Got a gist of a full log file if you add my twitter handle so I can DM you it 

Started in a squad of 4 - into main show 

Solo show logged correctly only on my elimination, however nothing after that point.

**Comment by amagus** [comment]
After doing some investigations, I found out that the new matchmaking system is using different messages whenever players are in parties or playing alone.
For instance, when you play alone, this is the line in the logs which is looked for:

> [StateMatchmaking] Begin matchmaking solo

But when you play with a party, the logs now print the following:

> [StateMatchmaking] Begin party communications

The code still uses the "Begin matchmaking" words to understand a new show is being played.

The solution I found is to look for "[StateMatchmaking] Begin" instead. There was some side effects on the party detection code, but easily fixable.

I will request a PR in order to try to fix this soon and link to this issue.
